core/iwasm/interpreter/wasm_loader.c

```diff
@@ -5764,6 +5776,8 @@ load_from_sections(WASMModule *module, WASMSection *sections,
     uint32 aux_heap_base_global_index = (uint32)-1;
     WASMFuncType *func_type;
     uint8 malloc_free_io_type = VALUE_TYPE_I32;
+    bool clone_const_strings = is_load_from_file_buf && !wasm_binary_freeable;
```

Comment:
Had better rename to `reuse_const_strings` or other? In `wasm_const_str_list_insert`, if `is_load_from_file_buf` is true, it will re-use the file buffer to adjust the const string, but not clone the string:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/f378edc408382b7bcf517b463d4a54948739075b/core/iwasm/interpreter/wasm_runtime.c#L4721-L4729